File: go/1018.go
package main

import "fmt"

func main() {
	var n int
	cedulas := [7]int{100, 50, 20, 10, 5, 2, 1}
	fmt.Scanf("%d", &n)
	fmt.Printf("%d\n", n)
	for _, c := range cedulas {
		fmt.Printf("%d nota(s) de R$ %d,00\n", n/c, c)
		n = n % c
	}
}

File: lua/1018.lua
local cedulas = {100, 50, 20, 10, 5, 2, 1}

local n = tonumber(io.read())
print(n)

for _, c in ipairs(cedulas) do
    print(string.format("%d nota(s) de R$ %d,00", math.floor(n / c), c))
    n = n % c
end
